modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/type/facet/portlet/TypeFacetPortletPreferencesImpl.java

```diff
@@ -77,15 +78,13 @@ public List<KeyValuePair> getAvailableAssetTypes(
 
 		String[] assetTypes = getAssetTypesArray();
 
-		String[] allAssetTypes = getAllAssetTypes(companyId);
-
 		if (ArrayUtil.isEmpty(assetTypes)) {
-			assetTypes = allAssetTypes;
+			return Collections.emptyList();
```

Comment:
Need to recheck this with regression here: https://github.com/liferay-core-infra/liferay-portal/pull/4319#issuecomment-1518221415
after reviewing the the code, I changed this part back to returning empty list.